File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    answer = 0
    astr = ''
    astr2 = ''
    for i in num_list:
        if i % 2 == 0:
            astr = astr + str(i)
            # print(astr)
        elif i % 2 == 1:
            astr2 = astr2 + str(i)
    answer = int(astr) + int(astr2)
    return answer

File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let answer1 = 0;
    let answer2 = 0;
    for(let i = 0; i < num_list.length; i++) {
        if(num_list[i] % 2 === 0) {
            answer1 += num_list[i].toString();
        } else if(num_list[i] % 2 === 1) {
            answer2 += num_list[i].toString();
        }
    }
    return parseInt(answer1) + parseInt(answer2);
}